downstream/assemblies/topologies/assembly-overview-tested-deployment-models.adoc

```diff
@@ -8,26 +8,27 @@ Red Hat tests {PlatformNameShort} {PlatformVers} with a defined set of topologie
 
 The following table outlines the different ways to install or deploy {PlatformNameShort}:
 
-
 .{PlatformNameShort} installation and deployment models
 [options="header"]
 |====
 | Mode | Infrastructure | Description | Tested topologies
 | RPM | Virtual machines and bare metal | The RPM installer deploys {PlatformNameShort} on {RHEL} by using RPMs to install the platform on host machines. Customers manage the product and infrastructure lifecycle.
 a| 
-* link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/tested_deployment_models/rpm-topologies#rpm-b-env-a[RPM enterprise topology]
-* link:{BaseURL}/red_hat_ansible_automation_platform/{PlatformVers}/html/tested_deployment_models/rpm-topologies#rpm-b-env-b[RPM mixed enterprise topology]
+* link:{URLTopologies}/rpm-topologies#rpm-a-env-a[RPM growth topology]
+* link:{URLTopologies}/rpm-topologies#rpm-a-env-b[RPM mixed growth topology]
```

Comment:
When I build the doc and click this link I'm brought to the section "RPM topologies" rather than "RPM mixed growth topology" is that right?

Reply:
Oh yep, I think that happens because the growth topologies are new and don't exist at docs.redhat.com yet.

Reply:
Cool, just wondering. Will merge now.